src/modules/core/components/Button/Button.css

```diff
@@ -75,6 +75,11 @@
   cursor: default;
 }
 
+.main:disabled:hover {
```

Comment:
Thats gonna change a lot of buttons!

Reply:
Yeah, but that was an issue everywhere, just nobody noticed it before.